einterfaces/mocks/AccountMigrationInterface.go

```diff
@@ -4,8 +4,10 @@
 
 package mocks
 
-import mock "github.com/stretchr/testify/mock"
-import model "github.com/mattermost/mattermost-server/v5/model"
+import (
```

Comment:
Not against this changes, but not related to the PR itself.

Reply:
These got changed when I ran `make einterfaces-mocks`, which was required for the new Cluster method that I added. So I can revert these changes manually, but ideally one should just run "make einterfaces-mocks" and commit whatever was changed. This probably happened because either someone did not run this or the tool changed itself in its latest tip, since we are just getting "go get -u".

Reply:
Yes, totally makes sense. Yes the idea of that is always keep the thing synchronized for everybody. In the next go version, I think we can pin the tools versions to avoid the `go get -u` every single time.

Reply:
Yep, that's https://github.com/mattermost/mattermost-server/issues/13088.

Reply:
Yes, exactly.